README.md

```diff
@@ -49,6 +49,22 @@ GitLens offers a wide range of features—here are the three most popular workfl
 
 - [**Streamline Collaboration**](#streamline-collaboration) &mdash; GitLens isn’t just for solo developers—it’s designed to enhance team collaboration. With Cloud Patches and Code Suggest, you can share and discuss suggested changes with any GitLens or GitKraken user, on multiple files and even PRs.
 
+## Home View - Your VS Code Workflow Hub
+
+Compact but powerful, the Home View lets you take your tasks and issues from code to merge. Start work on an issue or create PRs and new branches in one intelligent view. The perfect companion for developers looking to reduce tedious context switching and stay focused on their work in VS Code.
```

Comment:
I don't think we need "and new branches" here since that's part of the Start Work flow and we don't need to reiterate that GL helps you create normal branches.

```suggestion
Compact but powerful, the Home View lets you take your tasks and issues from code to merge. Start work on an issue and create PRs in one intelligent view. The perfect companion for developers looking to reduce tedious context switching and stay focused on their work in VS Code.
```